title_belt_nhl/tests/test_schedule.py

```diff
@@ -9,6 +9,7 @@
 
 MOCK_DATA_PATH = "./title_belt_nhl/tests/test_files/mock_league_schedule.json"
 MOCK_DATA_PATH_BIG = "./title_belt_nhl/tests/test_files/mock_league_schedule_big.json"
+MOCK_DATA_PATH_ALL = "./title_belt_nhl/tests/test_files/mock_league_schedule_all.json"
```

Comment:
Do we need the entire schedule or should we just add a few days to the `big` one?